lib/active_merchant/billing/gateways/braintree/token_nonce.rb

```diff
@@ -21,7 +21,7 @@ def url
       def create_token_nonce_for_payment_method(payment_method)
```

Comment:
I'm pretty sure this payment method needs to be updated to take in options because where would we be getting options[:merchant_account_id] from

Reply:
good catch, I added the changes :)